shared/core/types/transaction_test.go

```diff
@@ -75,47 +59,44 @@ func TestTransaction_ValidateBasic(t *testing.T) {
 	txNoNonce := proto.Clone(&tx).(*Transaction)
 	txNoNonce.Nonce = ""
 	er = txNoNonce.ValidateBasic()
-	require.Equal(t, ErrEmptyNonce().Code(), er.Code())
+	require.Error(t, er)
+	// require.Equal(t, ErrEmptyNonce().Code(), er.Code())
```

Comment:
Are we moving away from using `Error`? 

More here: https://github.com/pokt-network/pocket/pull/550/files#r1118743922